src/components/Form.jsx

```diff
@@ -0,0 +1,87 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import React, { useState } from 'react';
+// import { useDispatch } from 'react-redux';
+import { useNavigate } from 'react-router-dom';
+
+const Form = () => {
+  const navigate = useNavigate();
+  // const dispatch = useDispatch();
+  // const [name, setName] = useState('');
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+
+  // const handleChange = (e) => {
+  //   setName(e.target.value);
+  // };
+
+  const resetForm = () => {
+    setEmail('');
+    setPassword('');
+  };
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    fetch(
+      'http://localhost:3001/sessions',
+      {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify({
+          user: {
+            email,
+            password,
+          },
```

Comment:
I would suggest you change the email field and parameter to username as specified in the requirement.